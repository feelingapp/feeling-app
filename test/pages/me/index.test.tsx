import { shallow } from "enzyme"
import Me from "../../../src/pages/me"

describe("Me: ", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Me />)
    expect(wrapper).toMatchSnapshot()
  })
})
