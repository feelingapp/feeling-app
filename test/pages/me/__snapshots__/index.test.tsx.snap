// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Me:  matches snapshot 1`] = `
<Fragment>
  <Head>
    <title>
      Me / Feeling
    </title>
  </Head>
  <TopBar
    rightIcon={
      Object {
        "href": "/settings",
        "icon": <test-file-stub />,
        "label": "Settings",
      }
    }
    subtitle="How are you feeling today?"
    title="Hey Michael,"
  />
  <Container>
    <Calendar />
  </Container>
</Fragment>
`;
